backend/server/server.go

```diff
@@ -550,6 +552,44 @@ func NewServer(ctx context.Context, profile config.Profile) (*Server, error) {
 	}
 	recoveryUnaryInterceptor := recovery.UnaryServerInterceptor(recovery.WithRecoveryHandler(onPanic))
 	recoveryStreamInterceptor := recovery.StreamServerInterceptor(recovery.WithRecoveryHandler(onPanic))
+
+	interceptorLogger := func(l *slog.Logger) logging.Logger {
+		return logging.LoggerFunc(func(ctx context.Context, lvl logging.Level, msg string, fields ...any) {
+			var newFields []any
+			fieldsIter := logging.Fields(fields).Iterator()
+			for fieldsIter.Next() {
+				k, v := fieldsIter.At()
+				if slices.Contains(logging.SystemTag, k) || k == logging.ComponentFieldKey || k == logging.KindServerFieldValue || k == logging.MethodTypeFieldKey || k == "peer.address" {
+					continue
+				}
+				newFields = append(newFields, k, v)
+			}
+			l.Log(ctx, slog.Level(lvl), msg, newFields...)
+		})
+	}
+
+	loggingCodeToLevel := func(code codes.Code) logging.Level {
```

Comment:
@d-bytebase customizes the function for mapping our gRPC return codes to interceptor log level.
https://pkg.go.dev/github.com/grpc-ecosystem/go-grpc-middleware/v2@v2.0.0/interceptors/logging#WithLevels